action.yml

```diff
@@ -4,16 +4,42 @@ description: >
 branding:
   icon: check
   color: green
-runs:
-  using: docker
-  image: Dockerfile
-  entrypoint: "/entrypoint.sh"
-  post-entrypoint: "/cleanup.sh"
 inputs:
   args:
     description: Additional arguments to the sonar-scanner
     required: false
   projectBaseDir:
     description: Set the sonar.projectBaseDir analysis property
     required: false
-    default: .
+  scannerVersion:
+    description: Version of the Sonar Scanner CLI to use
+    required: false
+    default: 6.2.0.4584
```

Comment:
Can this be increased to `6.2.1.4610`, released on the 1st of October ([ref](https://github.com/SonarSource/sonar-scanner-cli/releases/tag/6.2.1.4610))?

Reply:
I have increased the version to `6.2.1.4610`, since `v3.0.0` of the action, released on Sept 24, is based on `sonar-scanner-cli:11.1` and latest release of 11.1 embeds 6.2.1.4610.